### PR TITLE
[NUI] Make all property setters in Shadow class to internal

### PR DESCRIPTION
**Pull request by rabbitfor**
### Description of Change ###
Instead in xaml, shadow can be a simple string such as,
"10.0 | #FF0000FFF", which means 10 blurRadius and red color.

### API Changes ###

 - ACR:

### CODE REVIEW

**Review by hinohie — APPROVED** [review]

File: src/Tizen.NUI/src/public/ViewProperty/Shadow.cs [hunk 1]
```diff
@@ -81,7 +90,7 @@ internal Shadow(PropertyMap propertyMap) : base(propertyMap)
         /// The color for the shadow.
         /// </summary>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public Color Color { get; set; }
+        public Color Color  { get; [InternalVisibleInXaml] internal set; }
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/ViewProperty/Shadow.cs:93]: delete space

File: src/Tizen.NUI/src/public/ViewProperty/Shadow.cs [hunk 2]
```diff
@@ -90,7 +99,7 @@ internal Shadow(PropertyMap propertyMap) : base(propertyMap)
         /// Negative value is ignored. (no blur)
         /// </remark>
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public float BlurRadius { get; set; }
+        public float BlurRadius  { get; [InternalVisibleInXaml] internal set; }
```
Comment (huiyueun) [inline src/Tizen.NUI/src/public/ViewProperty/Shadow.cs:102]: delete space

**Comment by huiyueun** [comment]
Is there any property which need to add this tag on existing property and apply it?
```csharp
[InternalVisibleInXaml]
```

File: src/Tizen.NUI/src/public/ViewProperty/Shadow.cs (hunk 1 above)
Comment (rabbitfor) [reply src/Tizen.NUI/src/public/ViewProperty/Shadow.cs]: Done!

File: src/Tizen.NUI/src/public/ViewProperty/Shadow.cs (hunk 2 above)
Comment (rabbitfor) [reply src/Tizen.NUI/src/public/ViewProperty/Shadow.cs]: Done!

**Comment by rabbitfor** [comment]
> Is there any property which need to add this tag on existing property and apply it?
> 
> ```cs
> [InternalVisibleInXaml]
> ```

I thought a lot about this `[InternalVisibleInXaml]`.
If a property setter is internal level, the API documentation indicates it is a readonly property.
But if we allow to set it in xaml, it's going to give a mess confusion to user so I decided to remove it.

**Pull request merged**